examples/reshape/app.tsx

```diff
@@ -0,0 +1,74 @@
+/** @jsx h */
+import { autocomplete } from '@algolia/autocomplete-js';
+import { createQuerySuggestionsPlugin } from '@algolia/autocomplete-plugin-query-suggestions';
+import { createLocalStorageRecentSearchesPlugin } from '@algolia/autocomplete-plugin-recent-searches';
+import { h, Fragment } from 'preact';
+import { pipe } from 'ramda';
+
+import '@algolia/autocomplete-theme-classic';
+
+import { groupBy, limit, uniqBy } from './functions';
+import { productsPlugin } from './productsPlugin';
+import { searchClient } from './searchClient';
+
+const recentSearchesPlugin = createLocalStorageRecentSearchesPlugin({
+  key: 'search',
+  limit: 10,
+});
+const querySuggestionsPlugin = createQuerySuggestionsPlugin({
+  searchClient,
+  indexName: 'instant_search_demo_query_suggestions',
+  getSearchParams() {
+    return {
+      hitsPerPage: 10,
+    };
+  },
+});
+
+const combineSuggestions = pipe(
```

Comment:
`combineSuggestions` doesn't really describe what this reshape function does, maybe something like `dedupe...`